app/models/cms/base_page.rb

```diff
@@ -23,4 +23,40 @@ def parent_sections
     sections.reverse
   end
 
+  def hidden?
+    published.nil?
+  end
+
+  def to_xml(options = {})
+    xml = options[:builder] || Nokogiri::XML::Builder.new
+
+    xml.__send__(options.fetch(:root, :page)) do |x|
+      unless new_record?
+        x.id id
+        x.created_at created_at.xmlschema
+        x.updated_at updated_at.xmlschema
+      end
+
+      x.title title
+      section.to_xml(builder: x, root: 'section', short: true) unless options[:short]
+      x.path(path) if respond_to?(:path)
+      if options[:short]
+        x.layout_name layout_name
+      else
+        layout&.to_xml(builder: x, short: true) || x.layout
+      end
+      x.system_name system_name
+      x.content_type content_type
+      x.liquid_enabled !liquid_enabled.nil?
```

Comment:
I think this is not always correct.
For example, if you check
```
pry(main)> CMS::Builtin::Page.find_by(system_name: 'login/new').liquid_enabled?
=> true
```

The API however returns
```
<liquid_enabled>false</liquid_enabled>
```

I guess this is due to some weird stuff on the model itself. Built-in pages should have the liquid enabled always set to true.

Reply:
Fixed